config/dcm2bids.json

```diff
@@ -153,6 +153,22 @@
                 "ImageType": ["*", "*", "P", "*", "*"],
                 "EchoNumber": "6"
             }
+        },
+        {
+            "dataType": "rfmap",
+            "modalityLabel": "rfmap_mag",
```

Comment:
can you check what is the current BIDS convention for naming B1 maps?

Reply:
@jcohenadad there's nothing on the [official specification](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html) right now but [here](https://bep001.readthedocs.io/en/dev/04-modality-specific-files/01-magnetic-resonance-imaging-data.html)'s a link that Agah shared with me some weeks ago where it is specified that RF field mapping maps should be named `TB1map`. I guess I can go with that for now event if it is not a definitive edition.